src/fixtures/configurations/azure-cost-with-schema-validation.json

```diff
@@ -0,0 +1,14 @@
+{
+  "componentId": "keboola.ex-azure-cost-management-sp",
+  "name": "schema-validation",
```

Comment:
Tu to nie je vidno, ale komponenta ma schemu pre config aj rows (API to vracia) ... a tak na nej mozeme neskor testovat validaciu configu.